lenskit-cli/src/main/java/org/lenskit/cli/commands/GlobalRecommend.java

```diff
@@ -25,16 +25,17 @@
 import net.sourceforge.argparse4j.impl.Arguments;
```

Comment:
I now know.

We should fold the code into `Recommend`, and have an option `--for-items` to recommend for items instead of users.